includes/definitions/stormshield.yaml

```diff
@@ -0,0 +1,16 @@
+os: stormshield
```

Comment:
The company name isn't usually the best name to use for the OS. Does the firmware / software have a name you can use? If not, the product genre should be used.